app/models/cases.py

```diff
@@ -25,16 +29,17 @@ class Case(TableModelMixin, table=True):
 
 class CaseRequest(BaseRequest):
     case_type: CaseTypes
-    notes: List[CaseNotesRequest] | None
-    people: List[PersonRequest] | None
-    case_tracker: CaseTrackerRequest | None
-    eligibility_outcomes: List[EligibilityOutcomesRequest] | None
+    # These fields all are optional for case create/update requests
+    notes: List[CaseNotesRequest] | None = None
+    people: List[PersonRequest] | None = None
```

Comment:
This is entirely my bad, but could you update the BasePerson class to read

```python

class BasePerson:
    name: str
    address: str | None
    phone_number: PhoneNumber | None = Field(default=None, sa_type=String, schema_extra={"json_schema_extra":{"example": "01632 960 001"}})
    postcode: Postcode | None = Field(default=None, sa_type=String, schema_extra={"json_schema_extra":{"example": "SW1A 1AA"}})
    email: EmailStr | None = Field(sa_type=String)
```

This adds an example to the Open API schema which fixes the issue where the Swagger docs example fails by default due to validation issues.

Reply:
updated to reflect comment, thank you.